Actions are failing now

Some incompatibility between numpy and pandas?

```
     gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DNPY_NO_DEPRECATED_API=0 -Ipandas/_libs/src/ujson/python -Ipandas/_libs/src/ujson/lib -Ipandas/_libs/src/datetime -I/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/numpy/core/include -I/opt/hostedtoolcache/Python/3.9.0/x64/include/python3.9 -c pandas/_libs/src/ujson/python/objToJSON.c -o build/temp.linux-x86_64-3.9/pandas/_libs/src/ujson/python/objToJSON.o -D_GNU_SOURCE -Wno-unused-function
    pandas/_libs/src/ujson/python/objToJSON.c: In function ‘initObjToJSON’:
    pandas/_libs/src/ujson/python/objToJSON.c:181:12: error: ‘NUMPY_IMPORT_ARRAY_RETVAL’ undeclared (first use in this function); did you mean ‘NPY_INOUT_ARRAY’?
         return NUMPY_IMPORT_ARRAY_RETVAL;
                ^~~~~~~~~~~~~~~~~~~~~~~~~
                NPY_INOUT_ARRAY
    pandas/_libs/src/ujson/python/objToJSON.c:181:12: note: each undeclared identifier is reported only once for each function it appears in
    pandas/_libs/src/ujson/python/objToJSON.c:182:1: warning: control reaches end of non-void function [-Wreturn-type]
     }
     ^
    error: command '/usr/bin/gcc' failed with exit code 1
```

https://github.com/pandas-dev/pandas/issues/34969

It seems that either `numpy 1.18` should be required or we should migrate to `pandas 1.0`.

--- a/.github/workflows/python.yml
+++ b/.github/workflows/python.yml
@@ -19,7 +19,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
       with:
-        python-version: '3.x'
+        python-version: '3.8'
     - name: Install pytest
       run: pip install pytest
     - name: Install zwad

